packages/marketplace/src/reducers/__stubs__/app-state.ts

```diff
@@ -12,6 +12,13 @@ const appState: ReduxState = {
       data: null,
       isAppDetailLoading: false,
     },
+    webComponent: {
```

Comment:
So here is the stub I was referring to above - can we extend this please if required?